imagersite/imagersite/templates/registration/activation_email.txt

```diff
@@ -0,0 +1,3 @@
+Thanks for registering, {{ user }}. Please click this link to activate your account.
+
+{{ site }}/registration/activate/{{ activation_key }}
```

Comment:
You should probably let the user know how many days they have until their activation key is invalid.